oml/metrics/triplets.py

```diff
@@ -9,50 +9,34 @@
 class AccuracyOnTriplets(IBasicMetric):
     metric_name = "accuracy_on_triplets"
 
-    def __init__(
-        self,
-        embeddings_key: str = "embeddings",
-        categories_key: Optional[str] = None,
-        categories_names_mapping: Optional[T_Str2Int_or_Int2Str] = None,
-    ):
+    def __init__(self, embeddings_key: str = "embeddings", categories_key: Optional[str] = None):
         """
 
         Args:
             embeddings_key: key to get embeddings which are tensors with the following structure:
                             [anchor, positive, negative, anchor, positive, negative...]
                              with the shape of [n_triplets * 3, features_dim]
             categories_key: key to get categories, which are integers or strings
-            categories_names_mapping: mapping from the category id to its name or vice versa
         """
-        if (categories_names_mapping is not None) and (categories_key is None):
-            raise ValueError(
-                "You have not specified category key but specified the mapping for " "the categories at the same time."
-            )
-
         self.embeddings_key = embeddings_key
         self.categories_key = categories_key
-        self.categories_names_mapping = categories_names_mapping
 
-        self.avg_online = OnlineAvgDict()
+        self.avg_online = defaultdict(OnlineAvgDict)  # type: ignore
 
     def setup(self, *args: Any, **kwargs: Any) -> None:  # type: ignore
-        self.avg_online = OnlineAvgDict()
+        pass
```

Comment:
add `self.avg_online = defaultdict(OnlineAvgDict) # type: ignore`

We need to flush avg for next epoch